blog/2025/01/the-satellite-toolbox-ecosystem.md

```diff
@@ -0,0 +1,235 @@
++++
+title = "The SatelliteToolbox.jl Ecosystem"
+authors = "Ronan Arraes Jardim Chagas"
+published = "3 January 2025"
+rss_pubdate = Date(2025, 01, 03)
+rss = """The history of the SatelliteToolbox.jl ecosystem"""
++++
```

Comment:
Why this format instead of the one used in previous posts:?

https://github.com/JuliaLang/www.julialang.org/blob/6ed315bbe2a0f5d88ff75256ed6810f814704eb6/blog/2025/01/this-month-in-julia-world.md?plain=1#L1-L5

Reply:
I just copied the one from here:

https://raw.githubusercontent.com/JuliaLang/www.julialang.org/refs/heads/main/blog/2024/10/julia-1.11-highlights.md

Do you want me to change it?

Reply:
Nope, that's fine.